server/Cargo.toml

```diff
@@ -22,6 +22,7 @@ rpassword = "7.3"
 serde.workspace = true
 sha2 = { version = "0.10", optional = true }
 tokio = { workspace = true, features = ["full"] }
+tokio-stream = "0.1"
```

Comment:
Do you need this dependency? for what?

Reply:
This is for `tokio_stream::StreamExt;`
I can't call `stream.try_next()` without this trait.